runtime/src/main/java/org/corfudb/runtime/collections/Index.java

```diff
@@ -46,28 +46,42 @@
      */
     public static class Spec<K, V, I> {
         private final Name name;
+        private final Name alias;
         private final MultiValueFunction<K, V, I> indexFunction;
 
         public Spec(Name name, Function<K, V, I> indexFunction) {
+            this(name, name, indexFunction);
```

Comment:
name will have the full path, correct?  dint we say the default will be only the last part of the path if alias is not specified?

Reply:
that is tru for protoBuf... however this can be extended by other Indexers, so this code is there to avoid a NPE in the case of other indexers for which the notion of alias is not present...